frontend/vesilahti/employee.tsx

```diff
@@ -607,6 +607,9 @@ const customizations: EmployeeCustomizations = {
     'DAYCARE',
     'TEMPORARY_DAYCARE',
     'PRESCHOOL_DAYCARE_ONLY',
+    'PREPARATORY',
+    'PREPARATORY_DAYCARE',
+    'PREPARATORY_DAYCARE_ONLY',
```

Comment:
There is only `PRESCHOOL_DAYCARE_ONLY` so i guess this should also be just `PREPARATORY_DAYCARE_ONLY`.

Reply:
PREPARATORY & PREPARATORY_DAYCARE removed